oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneDocumentMaker.java

```diff
@@ -283,8 +283,10 @@ protected boolean indexTypeOrderedFields(Document doc, String pname, int tag, Pr
             }
 
             if (f != null && includePropertyValue(property, 0, pd)) {
-                doc.add(f);
-                fieldAdded = true;
+                if (doc.getField(f.name()) == null) {
+                    doc.add(f);
+                    fieldAdded = true;
+                }
```

Comment:
Should we add a log in the else part here , which might prompt the developer to remove the duplicity form the index def ?